primitives/account/src/logs.rs

```diff
@@ -0,0 +1,364 @@
+use crate::Receipt;
```

Comment:
Mmm, it looks strange to me that we have this in `primitives/account` since this is not strictly related to accounts. Maybe consider moving it to one directory up (to `primitives/src`)?

Reply:
I think it makes sense to place it in the same place as receipts, they are reasonably similar. Another argument for letting it stay here is to that the wrapper structs that are within logs are return types of many of the account methods.